src/main/controllers/features/SoleTrader/typeOfBusinessController.ts

```diff
@@ -0,0 +1,47 @@
+import { NextFunction, Request, Response, Router } from "express";
+import { validationResult } from "express-validator";
+import * as config from "../../../config";
+import { FormattedValidationErrors, formatValidationError } from "../../../validation/validation";
+import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../../../utils/localise";
+import { SOLE_TRADER_TYPE_OF_BUSINESS, START, SOLE_TRADER_OTHER_TYPE_OFBUSINESS, SOLE_TRADER_ROLE } from "../../../types/pageURL";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
```

Comment:
Is there any point in making this a global variable instead of local to the get and post separately?